src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -3172,6 +3172,11 @@ private static void NativeCompletionProcessCommands(CompletionContext context, s
                         completionText = quote + completionText + quote;
                     }
 
+                    if (string.IsNullOrEmpty(listItemText))
+                    {
+                        continue;
```

Comment:
If we now silently skip empty strings should we document this?

Reply:
I'm surprised a process might not have a name - is this patching over a problem elsewhere, like in .Net?

At any rate, a comment on why this is expected would be useful.

Reply:
@lzybkr if I run PowerShell with `sudo`, the process names are there.  So presumably these are system processes where some properties are protected.  I'll add a comment.